### ISSUE TITLE
Generating a sequelize service backed by NeDB causes app to throw.

### ISSUE THREAD

**Issue by daffl** [feathersjs/feathers#953]
_From @eddyystop on August 10, 2017 12:48_

I added a Sequelize service and chose NeDB (!) as the database. The app threw when I started it.

```text
/home/johnsz/shared/repos/feathers-test/test-app/node_modules/sequelize/lib/sequelize.js:111
      options.dialect = urlParts.protocol.replace(/:$/, '');
                                         ^

TypeError: Cannot read property 'replace' of null
    at new Sequelize (/home/johnsz/shared/repos/feathers-test/test-app/node_modules/sequelize/lib/sequelize.js:111:42)
    at EventEmitter.module.exports (/home/johnsz/shared/repos/feathers-test/test-app/src/sequelize.js:6:21)
    at EventEmitter.configure (/home/johnsz/shared/repos/feathers-test/test-app/node_modules/feathers/lib/application.js:138:8)
```

The sequelize.model.js runs
```js
const sequelizeClient = app.get('sequelizeClient');
```
but that has not been created in `config/default.json`.

_Copied from original issue: feathersjs/cli#109_

**Comment by daffl** [comment]
_From @eddyystop on August 10, 2017 13:7_

Actually the database selection is messed up. All the DBs are offered in a seeminly endless list.

**Comment by daffl** [comment]
That's how inquirer does its lists. But you are right, the options list should be limited to valid databases.

**Comment by daffl** [comment]
_From @ruslankonev on January 16, 2018 14:43_

I have the same bug. What I need to do?

**Comment by daffl** [comment]
Choose one of the databases [Sequelize supports](http://docs.sequelizejs.com/).

**Comment by Asjamuar** [comment]
Locate sequelize.js and look for a line 
`const sequelize = new Sequelize(connectionString, {
    dialect: "mysql",
    logging: false,
    operatorsAliases: false,
    define: {
      freezeTableName: true
    }
  });`

Replace it with 
`const sequelize = new Sequelize(databaseName, user, password, {
    dialect: "mysql",
    logging: false,
    operatorsAliases: false,
    define: {
      freezeTableName: true
    }
  });`

**Comment by daffl** [comment]
Fixed in `@feathersjs/cli@4.0.0-pre.4`
